indico/modules/events/registration/templates/management/_print_badge_settings_form.html

```diff
@@ -1,15 +1,12 @@
 {% from 'forms/_form.html' import simple_form, form_fieldset, form_rows, form_row %}
 
-{% macro render_badge_print_settings_form(form, submit=_('Download PDF'), fields=none, templates=none, registrations=()) %}
-    {% if not fields %}
-        {% set fields = (
-            (none, ['template']),
-            (_('Page configuration'), ['save_values', 'dashed_border', 'page_size', 'page_orientation', 'page_layout'])
-        ) %}
+{% macro render_badge_print_settings_form(form, submit=_('Download PDF'), fieldsets=none, templates=none, registrations=()) %}
```

Comment:
Is the `fieldsets` arg still needed? It was you guys adding it (in #5428) and thus almost certainly nobody besides you using it - so if you solely rely on patching it on the form class level now, let's get rid of it again.

Reply:
Really? It is not needed for us anymore indeed, so let me remove that.

Reply:
Yes, I don't like having unused code that's very unlikely to be ever needed. So let's remove that arg from the macro and always use the data from the form